build-env-python/docker-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eo pipefail
+
+if [[ -n "$USER_UID" ]] && [[ -n "$USER_GID" ]]; then
+    # Create group and user only if they don't exist
+    [[ ! $(getent group builder) ]] && groupadd -r --gid "$USER_GID" builder
+    if [[ ! $(getent passwd builder) ]]; then
+        useradd --create-home --home-dir /home/builder --uid "$USER_UID" --gid "$USER_GID" --system --shell /bin/bash builder
+    usermod -aG wheel builder
```

Comment:
It's a detail, but the indentation needs to be corrected here

Reply:
Done !